usage/src/com/cloud/usage/UsageManagerImpl.java

```diff
@@ -373,6 +381,26 @@ protected void runInContextInternal() {
             }
 
             parse(job, startDate, endDate);
+            if (_runQuota){
+                try {
+                    _quotaManager.calculateQuotaUsage();
+                }
+                catch (Exception e){
+                    s_logger.fatal("Exception received while calculating quota " + e.getMessage());
+                    if (s_logger.isDebugEnabled()){
+                        e.printStackTrace();
+                    }
+                }
+                try {
+                    _alertManager.checkAndSendQuotaAlertEmails();
+                    _alertManager.sendMonthlyStatement(new Date());
+                } catch (Exception e) {
+                    s_logger.fatal("Exception received while sending alerts " + e.getMessage());
+                    if (s_logger.isDebugEnabled()) {
+                        e.printStackTrace();
```

Comment:
``e.printStackTrace()`` emits to the console.  Why aren't we passing the exception into the previous log statement?